samples/NativeModuleSample/cppwinrt/codegen/NativeAsyncMethodExamplesSpec.g.h

```diff
@@ -13,9 +13,17 @@
 
 namespace NativeModuleSample {
 
+REACT_STRUCT(AsyncMethodExamplesSpec_Response)
```

Comment:
While these changes may be necessary to align with the docs when the sample is updated to 0.73, it shouldn't really be a part of the docs cut.